### PR TITLE
Add Mega Stones

### PR DESCRIPTION
**Pull request by Ntonio36**
Sorry @AustinXII 🤷‍♂️ 😂

### CODE REVIEW

**Comment by AustinXII** [comment]
there should be more than just the items being added. Knock off calc's etc.

**Comment by Ntonio36** [comment]
The new commit also fixes #223 and #224, so make sure to close these issues on merge.

File: js/damage.js [hunk 1]
```diff
@@ -685,7 +686,6 @@ function getDamageResult(attacker, defender, move, field) {
 	var finalMod = chainMods(finalMods);
 
 	var damage = [];
-	var cumulatedStatDrops = 0;
```
Comment (AustinXII) [inline js/damage.js]: Did you mean for this line to be removed?

Comment (Ntonio36) [reply js/damage.js]: Yup, it was some sort of leftover variable I had forgotten to remove when I redid the multi-turn damage calc logic.
Don't worry though, this variable is used nowhere so we can peacefully get rid of it.

**Pull request merged**